Assets/Tests/InputSystem/CoreTests_Layouts.cs

```diff
@@ -691,6 +691,154 @@ public void TODO_Layouts_RegisteringMatcherForLayout_OverridesExistingMatchers()
         #endif
     }
 
+    [Preserve]
```

Comment:
It should be possible to just do _[assembly: Preserve]_ so that no stripping is done to the tests assembly.
Probabaly easier in the long run ;) 
https://docs.unity3d.com/Manual/ManagedCodeStripping.html

Reply:
This one is actually mainly there to make `API_TypesCreatedByReflectionHavePreserveAttribute` happy. Which, however, makes me wonder... why does that one even look at the test assemblies...